### PR TITLE
Feat: ✨ Add hook to customize Initiative deck preset

### PR DESCRIPTION
**Pull request by Stefouch**
This PR is a draft concept to add Hooks to configure the default Initiative Deck **before** it is created.

In the module's `Hooks.once('init')`, I have added the following call:
https://github.com/fvtt-fria-ligan/yearzero-combat-fvtt/blob/e45259e0ec445978fea4ce55bb20452b162abb9c/src/yze-combat.js#L48

Which expose a class with the following static function:
https://github.com/fvtt-fria-ligan/yearzero-combat-fvtt/blob/e45259e0ec445978fea4ce55bb20452b162abb9c/src/utils/client-hooks.js#L18-L22

So basically, any Game System can force the module to use their own JSON with the following in its main script:
```js
Hooks.on('yzeCombatInit', foo => foo.setSourceForInitiativeDeckPreset('path/to/my/initiative/deck.json'));
```

Not sure if it is the best way to do that, but it works!

Thoughts?

### CODE REVIEW

**Comment by aMediocreDad** [comment]
I'll add this to the combat tracker as well once we merge this one.

**Comment by Stefouch** [comment]
Anything else to put in this Hook?

If not, I think it's ready for review and merge
